doctordroid/README.md

```diff
@@ -0,0 +1,41 @@
+# Doctor Droid
+
+## Overview
+
+Doctor Droid is an observability analytics platform that analyzes alerts history and fetches metrics from upstream and downstream services. Use the integration to:
+- Decide better thresholds for alerts
+- Identify alerting coverage for your services
+- Enrich alerts for faster diagnosis
+
+## Setup
+
+### Installation
+1. Navigate to the [Doctor Droid integration tile][5] in Datadog.
```

Comment:
At this point, they are already on the tile so we can remove the first point. The information you include in the files of the PR is what will be on the tile :)